src/components/form.js

```diff
@@ -11,22 +12,28 @@ const Form = (props) => {
                 <div>
                     <label>First Name</label>
                     <input type="text" name="firstName" onChange={onChange}/>
+                    {firstName.length < 2 && firstName.length !== 0 ? <p style={{color:'red'}}>First Name must be at least two characters.</p>: null}
```

Comment:
You can use short-circuiting to return the `<p>` tag fi the expressions on the left are true. Instead of using the ternary you can do this.
 
```js
{firstName.length < 2 && firstName.length !== 0 && <p style={{color:'red'}}>First Name must be at least two characters.</p>}
```